health/notifications/alarm-notify.sh.in

```diff
@@ -3258,7 +3258,7 @@ Content-Transfer-Encoding: 8bit
                     </tr>
                     <tr>
                       <td align="left" style="font-size:0px;padding:10px 25px;padding-left:0;word-break:break-word;">
-                        <div style="font-family:Open Sans, sans-serif;font-size:14px;line-height:1.3;text-align:left;color:#35414A;">Discuss and troubleshoot with others on the Netdata <a href="https://community.netdata.cloud/" class="link" style="color: #00AB44; text-decoration: none;">community forums</a></div>
+                        <div style="font-family:Open Sans, sans-serif;font-size:14px;line-height:1.3;text-align:left;color:#35414A;">Join the troubleshooting discussion for this alert on our <a href="https://community.netdata.cloud/t/${name//[._]/-}" class="link" style="color: #00AB44; text-decoration: none;">community forums</a>.</div>
```

Comment:
Hello @burbuli8ra ,

I tested your PR and raised a personal alarm, and as expected the cloud does not have anything for it. As result I received the following message:

```
Oops! That page doesn’t exist or is private.
```

Before this PR the users were accessing the main page of our community. I am not sure that the best road is to redirect users for a page that does not exist :thinking: . Are we missing something in this PR?

An additional information, I did not test this with cloud, it was a local test.

Best regards!

Reply:
Hello @thiagoftsm ,

thank you for your feedback regarding the update of the url to community forums!
Actually, what you mention is a known issue to product and, as for now, they are ok to proceed with this and explore for a better handling of non existing urls in community forum, as a future enhancement!

The logic behind this update, is that we want the users to be more engaged when they receive a notification for an alert and be directly redirected to the specific topic of the alert they have received, instead of the generic community's homepage.

Kind regards!

Reply:
Hello,

I understood your explanation, I understand the goal, and I will approve your PR, but I think we should merge it only after we update the community error message.

Best regards!

Reply:
Thank you @thiagoftsm !

I will wait for a while to check if we can sync somehow with the team responsible for community to check if there is something that can be done from their part at this time!

Kind regads!

Reply:
Hi guys!

Yes, @burbuli8ra has done what the product expects. I will check though if there is an easy way to check if the alert is a custom one (either a completely new one, or a modified based on our stock alerts <- that would be more tricky to detect), and see if at least for completely new ones we should at least fallback to the generic link.

We could check if the `source` parameter to the script matches the `stock config directory`, and if it does we can use an general link to the community page.

Reply:
Hello @MrZammler !

After our sync with the alert managers, I will proceed with merging this PR when it receives a second approval, since we decided to first monitor the traffic of the community website to see how many 404 pages have been visited for the next 2-3 weeks and then revisit the best solution for custom alerts, if we see that an extra implementation for improving UX is actually needed! 😄